path_2023.py

```diff
@@ -58,6 +57,22 @@ def run() -> int:
         print("Error: upload_commands")
         return -1
     
+    # Get the current waypoint sequence
+    current_waypoint = drone.commands.next
```

Comment:
This should download the command sequence since the mission may be changed by something other than this program.